### DISCUSSION TITLE
error while using IfcConvert to glTF

### DISCUSSION THREAD

**Discussion by arunkvijay** [IfcOpenShell/IfcOpenShell#2770] [Q&A] [answered]
Hi, 
I am getting an error and dont see the option of converting to a .glb file. Converting to .dae format also doesnt work. 
I am using a conda environment with ifcopenshell installed. Is there any other way of doing this conversion to .glb

![image](https://user-images.githubusercontent.com/58535690/218481377-5a0e52da-cfbe-4a8b-88af-16a86974989f.png)

**Comment by Moult** [comment, marked as answer]
Recently gltf was missing from docker and was added https://github.com/IfcOpenShell/IfcOpenShell/commit/a701624d7ba00e3e0ef43e4c371aa96d3ab591ef https://github.com/IfcOpenShell/IfcOpenShell/commit/f912be78757cd7b5463267fc4634e43ec8764ba6 so perhaps it is also missing from Conda? Ping @Krande

**Comment by aothms** [comment]
It seems collada is off in both versions 

https://github.com/conda-forge/ifcopenshell-feedstock/blob/main/recipe/build.sh#L19

https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/conda/build.sh#L31
